sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -124,4 +124,76 @@ SELECT endswith('Spark SQL', 'QL');
 SELECT endswith('Spark SQL', 'Spa');
 SELECT endswith(null, 'Spark');
 SELECT endswith('Spark', null);
-SELECT endswith(null, null);
\ No newline at end of file
+SELECT endswith(null, null);
+
+-- to_number
```

Comment:
since we have extensive unit tests, we can only test some common cases in the end-to-end tests

Reply:
OK